src/VisualStudio/Core/Def/Implementation/InheritanceMargin/MarginGlyph/InheritanceMarginViewModel.cs

```diff
@@ -61,9 +61,13 @@ internal class InheritanceMarginViewModel
             ClassificationTypeMap classificationTypeMap,
             IClassificationFormatMap classificationFormatMap,
             InheritanceMarginTag tag,
-            double scaleFactor)
+            double zoomLevel)
         {
             var members = tag.MembersOnLine;
+
+            // ZoomLevel is 100 based. (e.g. 150%, 100%)
+            // ScaleFactor is 1 based. (e.g. 1.5, 1)
+            var scaleFactor = zoomLevel / 100;
```

Comment:
I move the zoom level -> scaleFactor conversion here so that this could be covered by the unit test change below